cgo_snappy_common.go

```diff
@@ -1,5 +1,5 @@
 package levigo
 
 // #cgo CFLAGS: -I${SRCDIR}/deps/snappy -DSNAPPY
-// #cgo CXXFLAGS: -I${SRCDIR}/deps/snappy -std=c++11 -g -O2 -fPIC -DPIC -DSNAPPY
+// #cgo CXXFLAGS: -I${SRCDIR}/deps/snappy -DPIC -DSNAPPY
```

Comment:
why no `-O2 -g` here (and in CFLAGS)?

Reply:
My understanding is that Cgo compiles all .c and .cc files in a package directory with the SAME set of compiler and linker flags. It generates them by concatenating all of them from all .go files. So this line is combined with the CXXFLAGS line from cgo_flags_common.go.

When I was testing this, I built the tests in this package with `go test -x .` and it showed me that there were duplicate flags being passed. This change resolves the duplicates.